warehouse/migrations/versions/4490777c984f_migrate_existing_data_for_release_is_.py

```diff
@@ -55,7 +55,7 @@ def upgrade():
                 """
             )
         )
-        conn.execute(sa.text("COMMIT"))
+        op.get_bind().commit()
```

Comment:
I don't thing we need to go back and change historical migrations at this point other than the minimum to get them working. I just wanted to note somewhere that I think in the future for this pattern, an auto commit block wrapping the `while` loop might end up being cleaner.